fast_llm/functional/config.py

```diff
@@ -91,3 +91,7 @@ class CrossEntropyImpl(str, enum.Enum):
     torch = "torch"
     fused = "fused"
     triton = "triton"
+
+class LossFunctionType(str, enum.Enum):
```

Comment:
Seems like a bad idea to call it a loss function. DPO doesn't just swap cross-entropy with another function, it changes the argument list altogether. It also affects data sampling and preprocessing.

How about we call it "training objective" instead or something like it?